pom.xml

```diff
@@ -2100,8 +2100,6 @@
             <configuration>
               <targetClasses>
                 <param>com.puppycrawl.tools.checkstyle.api.*</param>
-                <param>com.puppycrawl.tools.checkstyle.filefilters.*</param>
-                <param>com.puppycrawl.tools.checkstyle.filters.*</param>
```

Comment:
@Nimfadora , please recheck point from rnveach, I am merging your PR as we still raising Threshold, that is good to avoid leak in other PRs.